src/com/google/enterprise/adaptor/filenet/FileNetObjectFactory.java

```diff
@@ -30,6 +30,10 @@
  * Factory for producing instances various FileNet Objects.
  */
 class FileNetObjectFactory implements ObjectFactory {
+  @Override
+  public FileNetAdaptor.Traverser getTraverser(ConfigOptions options) {
```

Comment:
When we add the other traversers, you will either need to rename this add additional params.
How about getDocumentTraverser(options), or getTraverser("document", options)?